### ISSUE TITLE
@import placed above comment at beginning of output CSS

### ISSUE THREAD

**Issue by simonsmith** [sass/libsass#1080]
Given this `scss`:

``` sass
/** Some comment */

@import url("suitcss-components-grid");

.Tweet {
  clear: both;
  color: red;
}
```

I expect it to compile to:

``` css
/** Some comment */
@import url("suitcss-components-grid");
.Tweet {
  clear: both;
  color: red;
}
```

Which it does with Ruby Sass 3.4.12.

However libass 3.1.0 places the `@import` above the comment:

``` css
@import url("suitcss-components-grid");
/** Some comment */
.Tweet {
  clear: both;
  color: red; }
```

I would expect it to match the Ruby Sass output. I realise it's perhaps a minor issue but in my situation it effects some tooling that expects the comment on the first line.

Above tests were run in sassmeister.com

Thank you

**Comment by mgreter** [comment]
Ruby sass seems to preserve the order of appearance. Seems like we need to have a more generic `vector<Expression*> top_elements` instead of specific ones for each type (comment/import). Since this is not a regression I will assign it to 3.2.1.

```CSS
/** comment 1 */
@import url("import-1");
/** comment 2 */
@import url("import-2");
/** comment 3 */
foo { bar: baz; }
```

ruby sass:
```css
/** comment 1 */
@import url("import-1");
/** comment 2 */
@import url("import-2");
/** comment 3 */
foo {
  bar: baz;
}
```

libsass:
```CSS
@import url("import-1");
@import url("import-2");
/** comment 1 */
/** comment 2 */
/** comment 3 */
foo {
  bar: baz; }
```

**Comment by simonsmith** [comment]
Great, appreciate it

**Comment by xzyfer** [comment]
Spec added https://github.com/sass/sass-spec/pull/321

**Comment by xzyfer** [comment]
Fixed in [3.2.1](https://github.com/sass/libsass/releases/tag/3.2.1)
